runner_scripts/get_conan_broadcast_data.py

```diff
@@ -0,0 +1,71 @@
+import argparse
+import os
+import yaml
+
+def get_conan_broadcast_data(args):
+    with open("conandata.yml", "r") as f:
+        conan_data = yaml.safe_load(f)
+    version = conan_data["version"] if args.version == '' else args.version
+    project_name = args.project_name
+
+    if args.release == "true":
+        actual_version = version
+        user = "_"
+        channel = "_"
+        is_release_branch = True
+    else:
+        actual_version = version if "+" in version else f"{version}+{args.sha[:6]}"
+        user = args.user.lower()
+        ref_name = args.base_ref if args.event_name == "pull_request" else args.ref_name
+        if "beta" in version and args.event_name != "pull_request" and  ref_name == '.'.join(version.split('.')[:2]):
```

Comment:
Had to take a second to read the `'.'.join(version.split('.')[:2])` part
```suggestion
        [major, minor, _patch] = version.split('.')
        if "beta" in version and args.event_name != "pull_request" and  ref_name == f"{major}.{minor}"):
```